src/style.css

```diff
@@ -0,0 +1,75 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
+}
+
+.stars--0 .stars__star,
+.stars--1 .stars__star,
+.stars--2 .stars__star,
+.stars--3 .stars__star,
+.stars--4 .stars__star,
+.stars--5 .stars__star {
+  display: flex;
+  background-position: 50%;
+  background-repeat: no-repeat;
+  min-width: 16px;
+  min-height: 16px;
+}
+
+.stars--0 .stars__star {
+  background-image: url("images/star.svg");
+}
+
+.stars--1 .stars__star,
+.stars--2 .stars__star,
+.stars--3 .stars__star,
+.stars--4 .stars__star,
+.stars--5 .stars__star {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--1 .stars__star:nth-child(n+2),
+.stars--2 .stars__star:nth-child(n+3),
+.stars--3 .stars__star:nth-child(n+4),
+.stars--4 .stars__star:nth-child(n+5) {
+  background-image: url("images/star.svg");
+}
+
+.stars--0 .stars__star:not(:last-child),
+.stars--1 .stars__star:not(:last-child),
+.stars--2 .stars__star:not(:last-child),
+.stars--3 .stars__star:not(:last-child),
+.stars--4 .stars__star:not(:last-child),
+.stars--5 .stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+/* .stars__star {
+  height: 16px;
```

Comment:
Please remove the commented-out code block. It's always best to keep the codebase clean and free of unused code, as it might cause confusion in the future. 🧹